tests/10-api.spec.js

```diff
@@ -96,7 +97,7 @@ async function _setupApp() {
       getExpectedValues() {
         return {
           host: BASE_HOST,
-          rootInvocationTarget: [`${BASE_URL}/documents`]
+          rootInvocationTarget: [`${BASE_URL}/documents`, `${BASE_URL}/documents${QUERY}`]
```

Comment:
This doesn't seem like the right way to test this, but rather query-based attenuation should be used without modifying the root invocation target.

Reply:
resolved in https://github.com/digitalbazaar/ezcap-express/pull/29/commits/dab78e0700d91132c9e77a50286b49834ef4e44c